### PR TITLE
feat: create nl60

### PR DESCRIPTION
**Pull request by pierreboc**

### CODE REVIEW

**Review by moul — APPROVED** [review]

File: bertytech/content/newsletter/news-60/index.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+aliases: ["/newsletter/weekly-60"]
+---
+
+# News #60
+
+Hi cool kids 🏴☠️
+
+Here's a newsletter that will make you happy: Berty is on TestFlight and will be soon on the Play Store.🤩🤩🤩🤩. It's coming soon. We know we announced in our previous newsletter that It was supposed to arrive last week. But there is a little delay in the validation and some bugs to fix. Anyway, if you have an iPhone, you can download the alpha! (Read below how!)
+
+Just a quick word about the team before letting you dive into this newsletter: everyone is doing well. We admit that the restaurants and bars are finally open again until 11pm. It feels so good to be back to normal. We can't wait to celebrate the release of our Alpha community. We are very excited to showcase our work. 😊
```
Comment (aeddi) [inline bertytech/content/newsletter/news-60/index.md:13]: `We admit that the restaurants and bars are finally open again until 11pm.`

What the meaning of that? A word is missing maybe?

File: bertytech/content/newsletter/news-60/index.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+## 🚀 Open Source
+
+### Berty Alpha Community
+    Alpha is ONLY available on iOS (TestFlight) for now.
+    Android version is coming soon.
+    Join our Discord to be notified. 
+
+**Why a community alpha version?**
+
+We decided an app version made to open doors to developers and testers, and to contribute on the development. Please note that the app will host data/metadata to help developers to go forward during the Alpha version. It’s only a TEST version to allow community to have testing sessions, explore code, UX/UI, and improve all kinds of bugs.
```
Comment (aeddi) [inline bertytech/content/newsletter/news-60/index.md:25]: `We decided an app version` -> word missing?

Comment (aeddi) [reply bertytech/content/newsletter/news-60/index.md:25]: `to help developers to go forward during the Alpha version.` -> `to help developers identify and fix bugs during the Alpha version.`

File: bertytech/content/newsletter/news-60/index.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+Our goal with the Alpha Community: 
+* Every community members can download the Alpha
+* This version is not ready for daily usage, it’s just a Alpha version to discover the Berty app.
```
Comment (aeddi) [inline bertytech/content/newsletter/news-60/index.md:29]: `a Alpha version` -> `an Alpha version`

File: bertytech/content/newsletter/news-60/index.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+* TOR feature is not ready yet. IPs are not hidden.
+* Account Persistence : not ready yet, your data will be deleted for next updates
```
Comment (aeddi) [inline bertytech/content/newsletter/news-60/index.md:31]: `your data will be deleted for next updates` -> `your data will probably be lost during updates`

File: bertytech/content/newsletter/news-60/index.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+* Remove crashes, white screens, or app freeze
```
Comment (aeddi) [inline bertytech/content/newsletter/news-60/index.md:32]: `screens, or app freeze` -> `screens and app freezes`

File: bertytech/content/newsletter/news-60/index.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+* Remove crashes, white screens, or app freeze 
+
+Ready?
+
+
+![image alt](https://media.giphy.com/media/XHjniwBKPxhnykNYno/source.gif)
+
+**How to download?** 
+
+Join us on [Discord](https://crpt.fyi/berty-discord), check out our annoucement channel to get all the info to download the alpha. And if you have an Android phone, we are as impatient as we are. It's a matter of days now.
```
Comment (aeddi) [inline bertytech/content/newsletter/news-60/index.md:41]: `we are as impatient as we are` -> `we are as impatient as you are to make it available`

**Review by aeddi — CHANGES_REQUESTED** [review]

File: bertytech/content/newsletter/news-60/index.md (hunk 1 above)
Comment (aeddi) [reply bertytech/content/newsletter/news-60/index.md]: `Play Store.🤩🤩🤩🤩.` -> `Play Store 🤩🤩🤩🤩.`

**Comment by pierreboc** [comment]
@aeddi thanks! fixed it all in this commit: https://github.com/berty/www.berty.tech/pull/278/commits/2a3c614bf14535bab1813da2bed232220b292e2d

File: bertytech/content/newsletter/news-60/index.md (hunk 1 above)
Comment (aeddi) [reply bertytech/content/newsletter/news-60/index.md]: `that It was supposed` -> `that it was supposed`

Comment (aeddi) [reply bertytech/content/newsletter/news-60/index.md]: `download the alpha` -> `download the Alpha` (consistency)

File: bertytech/content/newsletter/news-60/index.md (hunk 2 above)
Comment (aeddi) [reply bertytech/content/newsletter/news-60/index.md]: `the app will host data/metadata` -> `the app will send data/metadata on third-party servers`

Comment (aeddi) [reply bertytech/content/newsletter/news-60/index.md]: `UX/UI, and improve all kinds of bugs.` -> `UX/UI and fix all kinds of bugs.`

**Comment by aeddi** [comment]
> @aeddi thanks! fixed it all in this commit: [2a3c614](https://github.com/berty/www.berty.tech/commit/2a3c614bf14535bab1813da2bed232220b292e2d)

Added other comments in the meantime, I will review this commit and copy them so it will easier to fix.

File: bertytech/content/newsletter/news-60/index.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+title: "Berty News #60"
+date: 2021-06-11
+aliases: ["/newsletter/weekly-60"]
+---
+
+# News #60
+
+Hi cool kids 🏴☠️
+
+Here's a newsletter that will make you happy: Berty is on TestFlight and will be soon on the Play Store.🤩🤩🤩🤩. It's coming soon. We know we announced in our previous newsletter that It was supposed to arrive last week. But there is a little delay in the validation and some bugs to fix. Anyway, if you have an iPhone, you can download the alpha! (Read below how!)
```
Comment (aeddi) [inline bertytech/content/newsletter/news-60/index.md:11]: `Play Store.🤩🤩🤩🤩.` -> `Play Store 🤩🤩🤩🤩.`

Comment (aeddi) [reply bertytech/content/newsletter/news-60/index.md:11]: `the alpha!` -> `the Alpha!` (consistency)

File: bertytech/content/newsletter/news-60/index.md [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+Just a quick word about the team before letting you dive into this newsletter: everyone is doing well. Restaurants and bars are finally open again until 11pm. It feels so good to be back to normal. We can't wait to celebrate the release of our Alpha community. We are very excited to showcase our work. 😊
+
+
+## 🚀 Open Source
+
+### Berty Alpha Community
+    Alpha is ONLY available on iOS (TestFlight) for now.
+    Android version is coming soon.
+    Join our Discord to be notified.
```
Comment (aeddi) [inline bertytech/content/newsletter/news-60/index.md:21]: `to be notified.` -> `to be notified of any news about the Alpha.`

File: bertytech/content/newsletter/news-60/index.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+## 🚀 Open Source
+
+### Berty Alpha Community
+    Alpha is ONLY available on iOS (TestFlight) for now.
+    Android version is coming soon.
+    Join our Discord to be notified. 
+
+**Why a community alpha version?**
+
+We decided a community version to open doors to developers and testers, and to push forward on the development. Please note that the app will host data/metadata to help developers identify and fix bugs during the Alpha version. It’s only a TEST version to allow community to have testing sessions, explore code, UX/UI, and improve all kinds of bugs.
```
Comment (aeddi) [inline bertytech/content/newsletter/news-60/index.md:25]: `We decided a community version to open doors`
A word is missing?

Comment (aeddi) [reply bertytech/content/newsletter/news-60/index.md:25]: `Please note that the app will host data/metadata` -> `Please note that the app will send data/metadata to third party servers`

Comment (aeddi) [reply bertytech/content/newsletter/news-60/index.md:25]: `UX/UI, and improve all kinds of bugs.` -> `UX/UI and fix all kinds of bugs.`

**Review by aeddi — CHANGES_REQUESTED** [review]

**Review by aeddi — APPROVED** [review]
LGTM 👍

**Pull request merged**